lib/src/functions.dart

```diff
@@ -151,8 +151,8 @@ Future<void> screenLockCreate({
   SecretsConfig? secretsConfig,
   KeyPadConfig? keyPadConfig,
   DelayBuilderCallback? delayBuilder,
-  Widget? customizedButtonChild,
-  VoidCallback? customizedButtonTap,
+  Widget Function(bool confirmed)? customizedButtonChild,
+  Function(bool confirmed, InputController controller)? customizedButtonTap,
```

Comment:
This is missing a return type for the Function. 
```suggestion
  void Function(bool confirmed, InputController controller)? customizedButtonTap,
```